database/migrations/wallet-migration.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+      await queryInterface.createTable('wallets', 
+      {
+        id: {
+          type: Sequelize.UUID,
+          primaryKey: true,
+        },
+        user_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+        status: {
+            type: Sequelize.ENUM('Active', 'Inactive'),
+            allowNull: false,
+          },
+        balance: {
+            type: Sequelize.DECIMAL,
```

Comment:
the type of the decimal should be defined to reflect the number of decimal places and whole numbers that it should accept in this case. it should also be unsigned since the user cannot have a negative balance
``` type: Sequelize.DECIMAL(20, 2).UNSIGNED ```